webpack.config.js

```diff
@@ -78,6 +78,7 @@ const config = {
   entry: {
     index: path.join(__dirname, "src", "index.js"),
     hub: path.join(__dirname, "src", "hub.js"),
+    link: path.join(__dirname, "src", "link.js"),
     "avatar-selector": path.join(__dirname, "src", "avatar-selector.js")
```

Comment:
IIUC, can this split into another bundle? that possible?

Reply:
this creates a new landing page and bundle for `link.html`, yep